### PR TITLE
@alloy => [v2] Finish artist app

### PR DESCRIPTION
**Pull request by mzikherman**
All tests and type check should be good to go.

### CODE REVIEW

File: src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx [hunk 1]
```diff
@@ -27,4 +29,4 @@ export const ArtistHeaderFixture = {
     cv: true,
     auction_lots: true,
   },
-}
+} as ArtistHeader_Test_QueryRawResponse["artist"]
```
Comment (alloy) [inline src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx]: FYI It’s better to move this to the variable definition, I think TS will give you better info.

**Comment by alloy** [comment]
Nice!

File: src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx (hunk 1 above)
Comment (mzikherman) [reply src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx]: It's actually there too! This one is unneeded 🙈

Comment (alloy) [reply src/Apps/__tests__/Fixtures/Artist/Components/ArtistHeader.tsx]: 😸

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**